Platform Documentation.md

```diff
@@ -854,7 +854,7 @@ Connection to ssh.cloudcontrolled.net closed.
 
  * Stacks define the common runtime environment.
  * They are based on Ubuntu and stack names match the Ubuntu release's first letter.
- * Luigi supports only PHP. Pinky supports multiple languages according to the available [buildpacks](#buildpacks-and-the-procfile).
+ * Pinky supports multiple languages according to the available [buildpacks](#buildpacks-and-the-procfile).
```

Comment:
Without Luigi I think we can remove the whole bullet point. All future stacks will also support all languages.